evap/evaluation/models.py

```diff
@@ -1599,6 +1599,10 @@ def send_contributor_publish_notifications(cls, evaluations, template=None):
 
         evaluations_per_contributor = defaultdict(set)
         for evaluation in evaluations:
+            # all responsibles unconditionally get a notification
```

Comment:
we shouldn't send an email if no results are published, only in the if and elif cases below. i'm sorry that the issue description is not precise enough.